esphome/components/mcp4728/__init__.py

```diff
@@ -0,0 +1,28 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c
+from esphome.const import CONF_ID
+
+DEPENDENCIES = ["i2c"]
+MULTI_CONF = True
+AUTO_LOAD = ["output"]
+CODEOWNERS = ["@brotherdust"]
+
+mcp4728_ns = cg.esphome_ns.namespace("mcp4728")
+MCP4728Output = mcp4728_ns.class_("MCP4728OutputComponent", cg.Component, i2c.I2CDevice)
```

Comment:
Just for less confusion, this should match the c++ class name
```suggestion
MCP4728OutputComponent = mcp4728_ns.class_("MCP4728OutputComponent", cg.Component, i2c.I2CDevice)
```